parser/parser.y

```diff
@@ -4527,12 +4528,25 @@ ShowTableAliasOpt:
 	}
 
 FlushStmt:
-	"FLUSH" NoWriteToBinLogAliasOpt TableOrTables TableNameListOpt WithReadLockOpt
+	"FLUSH" NoWriteToBinLogAliasOpt FlushOption
+	{
+		tmp := $3
+		if x, ok := tmp.(*ast.FlushTableStmt); ok {
```

Comment:
You can refer to mysql's parser. It does not check if it is FlushTableStmt.
https://github.com/mysql/mysql-server/blob/5.7/sql/sql_yacc.yy#L12350